stories/module-analytics-components.stories.js

```diff
@@ -317,7 +317,7 @@ generateAnalyticsWidgetStories( {
 
 		},
 	] ),
-	Component: DashboardUniqueVisitorsWidget,
+	Component: DashboardSearchVisitorsWidget,
```

Comment:
We also need to update report options here to replicate changes in the component.